### ISSUE TITLE
[Discussion] Do we have presto native worker deployment guide?

### ISSUE THREAD

**Issue by xumingming** [prestodb/presto#21242]
I know there is a DEVELOPMENT guide for presto native worker(https://github.com/prestodb/presto/tree/master/presto-native-execution), do we have DEPLOYMENT guide for deploying presto native worker in linux production environment?

**Comment by aditi-pandit** [comment]
@xumingming : We do not. There is some experience related to production deployments at Meta,  Bytedance that will be presented at PrestoCon Dec 2023. We can consolidate the presentations to a writeup for the github/site. 

Would be great if you could outline more specific questions you have about deployment (here or in other issues), and we can consolidate those as well.

**Comment by xumingming** [comment]
@aditi-pandit  Do not have very specific questions now, was hoping there is a simple step-by-step doc I can follow to setup a presto native worker cluster for future test like perf test.

**Comment by aditi-pandit** [comment]
@xumingming : You can start with setting up a native worker cluster using docker-compose from  https://github.com/prestodb/prestorials/tree/main/docker-compose-native

**Comment by xumingming** [comment]
@aditi-pandit Thanks, I will give it a try.

**Comment by xumingming** [comment]
@aditi-pandit I have specific questions now :(

I didn't fully follow the docker compose guide(https://github.com/prestodb/prestorials/tree/main/docker-compose-native), but the docker compose guide & the development guide did give me some hints, I have done the following:

1. I started a Presto Java Coordinator successfully.
2. I started a native worker with `discovery.uri` pointing to the java coordinator(`http://127.0.0.1:8080`) with the command:

```
./presto_server --logtostderr=1 --v=1 --velox_exception_user_stacktrace_enabled=true
```

The native worker seems started successfully, because I saw the following in stdout:

```
1028 21:29:46.791427   451 PeriodicServiceInventoryManager.cpp:99] Announcement succeeded: HTTP 202
```

But from Presto UI, the worker count does not include the native worker, there is only one worker(the java coordinator itself.), What could be the problem?
